manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: argo.dh-prod-argo
+spec:
+  destination:
+    namespace: dh-prod-argo
+    server: https://datahub.psi.redhat.com:443
+  project: data-hub
+  source:
+    path: argo/overlays/dh-prod-argo
+    repoURL: https://github.com/AICoE/idh-manifests.git
+    targetRevision: production
+  syncPolicy:
```

Comment:
Could we disabled auto-sync initially? Just until we have ArgoCD fully managing things.

Reply:
You mean, disable auto-sync in prod and have it enabled for stage only? That make sense. :+1: Will do. :slightly_smiling_face: